packages/transactions/src/decompile-transaction.ts

```diff
@@ -1,10 +1,11 @@
 import { Address, assertIsAddress } from '@solana/addresses';
 import { pipe } from '@solana/functional';
-import { AccountRole, IAccountMeta, IInstruction } from '@solana/instructions';
+import { AccountRole, IAccountLookupMeta, IAccountMeta, IInstruction } from '@solana/instructions';
 import { SignatureBytes } from '@solana/keys';
 
 import { Blockhash, setTransactionLifetimeUsingBlockhash } from './blockhash';
 import { CompilableTransaction } from './compilable-transaction';
+import { getCompiledAddressTableLookups } from './compile-address-table-lookups';
```

Comment:
I'm not sure if it does a whole lot, but you can `import type` this.

Reply:
Thanks good spot! We should probably lint for this somehow